README.md

```diff
@@ -586,6 +587,38 @@ func main() {
 }
 ```
 
+#### Values from Files
+
+You can also have the default value set from file via `FilePath`.  e.g.
+
+<!-- {
+  "args": ["&#45;&#45;help"],
+  "output": "language for the greeting.*APP_LANG"
```

Comment:
You can find more details about `gfmrun` here: https://github.com/urfave/gfmrun#go